runtime/codert_vm/module.xml

```diff
@@ -59,10 +59,10 @@
 			<include path="j9gcinclude"/>
 			<include path="$(OMR_DIR)/gc/include" type="relativepath"/>
 
-			<include path="../tr.source/runtime" type="relativepath"/>
-			<include path="../tr.source/codegen" type="relativepath"/>
-			<include path="../tr.source/base" type="relativepath"/>
-			<include path="../tr.source/trj9/runtime" type="relativepath"/>
+			<include path="../compiler/runtime" type="relativepath"/>
+			<include path="../compiler/codegen" type="relativepath"/>
+			<include path="../compiler/base" type="relativepath"/>

```

Comment:
The above 3 paths no longer exist in OpenJ9 and can be deleted.

Reply:
Fixed in 626683a